stories/index.stories.js

```diff
@@ -926,6 +929,13 @@ storiesOf('Search Components/SearchBox', module)
 		components: { BaseSearchBox },
 		template: '<base-search-box :subProps="{ showIcon, showFilter: false}"/>',
 	}))
+	.add('with showDistinctSuggestions', () => ({
```

Comment:
@SavvyShah The order of the stories should be maintained, let's have all the document suggestions stories right after the last added story, which in this case is the [compoundClause story](https://github.com/appbaseio/vue-playground/blob/75de207395a6a51e763d4c8dff022887cbd50e53/stories/index.stories.js#L1239).